src/Components/Web.JS/src/Platform/WebAssemblyProgressReporter.ts

```diff
@@ -0,0 +1,21 @@
+import { WebAssemblyProgressService } from "./WebAssemblyProgressService";
+
+export class WebAssemblyProgressReporter {
+    static setProgress(resourcesTotal: number, resourcesLoaded: number): void {
+        const circle = document.querySelector('.progress') as SVGCircleElement;
```

Comment:
Let's use an element id here (instead of a css class) to uniquely identify the circle we want;

```suggestion
        const circle = document.getElementById('progress') as SVGCircleElement;
```